nemo/utils/export_utils.py

```diff
@@ -263,7 +282,7 @@ def replace_FusedLayerNorm(n: nn.Module) -> Optional[nn.BatchNorm2d]:
         if isinstance(n, FusedLayerNorm) or isinstance(n, MixedFusedLayerNorm):
             mod = nn.LayerNorm(n.normalized_shape, eps=n.eps, elementwise_affine=n.elementwise_affine,).to(dev)
         elif isinstance(n, FastLayerNorm):
-            mod = nn.LayerNorm(n.weight.shape, eps=n.epsilon, elementwise_affine=True, dtype=torch.float16,).to(dev)
+            mod = nn.LayerNorm(n.weight.shape, eps=n.epsilon, elementwise_affine=True, dtype=torch.float,).to(dev)
```

Comment:
Should we keep original dtype here ?

Reply:
Also, as part of general export process, we currently guard LayerNorm with fp32 casts. So this won't have any effect. If in fact protecting LayerNorm from autocast is not necessary - we should revise that part as well.